src/test/resources/com/puppycrawl/tools/checkstyle/checks/sizes/linelength/InputSimple.java

```diff
@@ -0,0 +1,225 @@
+////////////////////////////////////////////////////////////////////////////////
+// Test case file for checkstyle.
+// Created: Feb-2001
+// Ignore error
+////////////////////////////////////////////////////////////////////////////////
+package com.puppycrawl.tools.checkstyle.checks.sizes.linelength;
+
+/**
+ * Contains simple mistakes:
+ * - Long lines
+ * - Tabs
+ * - Format of variables and parameters
+ * - Order of modifiers
+ * @author Oliver Burn
+ **/
+final class InputSimpleLineLength
```

Comment:
Please explain in more detail
I have a resource file - InputSimple.java. It must be renamed to InputSimpleLineLength?
Or it is rename LineLengthCheckTest to rename on InputLineLengthCheckTest?